source/server-selection/server-selection-tests.rst

```diff
@@ -294,22 +296,37 @@ Multi-threaded and async drivers MUST also implement the following prose test:
             blockTimeMS: 500,
             appName: "loadBalancingTest",
         },
-    }
+     }
 
-3. Create a client with both mongoses' addresses in its seed list,
-   appName="loadBalancingTest", and command monitoring enabled.
+3. Insert an empty document into the collection being used for the test.
 
-4. Start 10 concurrent threads / tasks that each run 10 `findOne` operations
-   using that client.
+4. Create a client with both mongoses' addresses in its seed list,
+   appName="loadBalancingTest", and localThresholdMS=30000.
 
-5. Assert that fewer than 25% of the CommandStartedEvents occurred on the mongos
-   that the failpoint was enabled on.
+   * localThresholdMS is set to a high value to help avoid a single mongos
+     being selected too many times due to a random spike in latency in the
+     other mongos.
 
-6. Disable the failpoint.
+5. Using CMAP events, ensure the client's connection pools for both
+   mongoses have been saturated, either via setting minPoolSize=maxPoolSize or
+   executing operations.
 
-7. Repeat this test without any failpoints and assert that each mongos was
-   selected roughly 50% (within +/- 10%) of the time.
+   * This helps reduce any noise introduced by connection establishment latency
+     during the actual server selection tests.
 
+6. Start 10 concurrent threads / tasks that each run 10 `findOne` operations
```

Comment:
.Net implementation test got stable only at 100 iterations, should we increase the number of iterations or mention that drivers might need to fine-tune it?

Reply:
Sorry, just for the second test, missed that.